### ISSUE TITLE
Unable to find Java test report after the test completion

### ISSUE THREAD

**Issue by sevapopov2** [microsoft/vscode-java-test#1387]
I will be glad if the issue can be closed and if there is the solution to it that already exists, but here's the problem I have:
After some versions of VSCode or test runner plugin, I am not sure, java test report web page stopped displaying after tests with failures are completed. I am screen reader user, this page really helped me in viewing the test results and understanding what went wrong quickly. However, I cannot do that quickly now. Test pick output option allows to view the test results in the console, but it's not as even comfortable as viewing it on a web page. So I don't know how to view test results, what tests passed, what failed in a comfortable way as for a Screen reader user.
Steps to reproduce:
1. Open any Java project with JUnit tests. Make sure that 1 or more tests are failed.
2. Run tests.
Expected result:
After the tests finish executing, there should be a java test report web page opened in a separate VSCode tab where we are able to see the report structured on web page with buttons for passed and failed tests visibility.
Actual result:
Nothing is displayed after the tests execution, so I am not able to view the test results in a comfort way.
Reproduced on:
Windows 11 x64 bit, NVDA 2021.3.3
Arch Linux, Orca 42RC Master branch.
VSCode version:
Windows:
Version: 1.64.2 (user setup)
Commit: f80445acd5a3dadef24aa209168452a3d97cc326
Date: 2022-02-09T22:02:28.252Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000
On Arch Linux VSCode Insiders is used.
Test Runner plugin version: 0.34.1.
I will be very glad if there is something that can be done about it, or even better if I simply need to turn on some settings to make it work again.
Thank you!

**Comment by jdneo** [comment]
Last year we adopted the VS Code's new testing API, with that change the web view page now becomes a editor peek view, which shows the test messages.

@connor4312 Do you know if there is any guidance to help users read the test results via tools like NVDA?

**Comment by sevapopov2** [comment]
Hello, thank you for your answer! How can I choose this editor pick view? What steps should I perform? I may check and report if it's accessible or not and create an issue for that if needed. Thank you!

**Comment by jdneo** [comment]
By default, the peek view will automatically open when there are cases failed. You can control the behavior via the setting `testing.automaticallyOpenPeekView`.

And you can leverage the command `Test: Peek Output` to open the peek view.

**Comment by sevapopov2** [comment]
Thank you for your answer! I will check this option, I remember that I saw it in the documentation but couldn't find it in the command palet. I will try again. But what I can tell now is that after the test is executed, if there is a failure, screen reader automatically starts reading the error message but the focus is placed in the editor with the code.

**Comment by jdneo** [comment]
> after the test is executed, if there is a failure, screen reader automatically starts reading the error message but the focus is placed in the editor with the code.

@connor4312 Maybe this is an accessibility issue which can be fixed at VS Code side?

**Comment by jdneo** [comment]
Sounds like might be an issue of VS Code. since we now adopted the testing API. The UX are mostly controlled by VS Code. Would you mind me to migrate this issue to the VS Code repo and we can discuss there?

**Comment by sevapopov2** [comment]
Yes that will be absolutely fine. Maybe we can figure out if it's really an issue or I misunderstand something.

**Comment by jdneo** [comment]
Filed a new issue here: https://github.com/microsoft/vscode/issues/145298

**Comment by jdneo** [comment]
I'll close this issue since it's tracked by upstream. Thank you for reporting it @sevapopov2

**Comment by sevapopov2** [comment]
Hello.
Sorry for taking so much time answering to question if everything is fixed in [this issue](https://github.com/microsoft/vscode/issues/145298).
Now I checked and since I cannot comment in the issue I linked above, I will comment here.
Everything is fixed now and when pressing CTRL+; and CTRL+m command focus moves to test results tree.
Thank you for fixing the issue and sorry for so late reply again!
